patches/media-gpu-vaapi-vaapi_picture_tfp.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/media/gpu/vaapi/vaapi_picture_tfp.cc b/media/gpu/vaapi/vaapi_picture_tfp.cc
+index 227c31b18bef0c677c0f8e9e9206cab146dabc75..b42620d74553e98f18eb4b72eddbea623b596fdf 100644
+--- a/media/gpu/vaapi/vaapi_picture_tfp.cc
++++ b/media/gpu/vaapi/vaapi_picture_tfp.cc
+@@ -57,7 +57,7 @@ bool VaapiTFPPicture::Initialize() {
+   if (make_context_current_cb_ && !make_context_current_cb_.Run())
+     return false;
+ 
+-  glx_image_ = new gl::GLImageGLX(size_, GL_RGB);
++  glx_image_ = new gl::GLImageGLX(size_, gfx::BufferFormat::BGRX_8888);
```

Comment:
how did this issue come up? It's merged in 82, but did something break for you on Linux that prompted pulling it into 81?

Reply:
We are building with vappi enabled, which is different from Chromium's default. I was getting build errors.

Reply:
what was the build error?

Reply:
oh, this is an upstream patch? That's fine then